src/components/MessageBox.js

```diff
@@ -0,0 +1,91 @@
+const React = require('react');
+const _merge = require('lodash/merge');
+
+const StyleConstants = require('../constants/Style');
+const Icon = require('../components/Icon');
+
+const MessageBox = React.createClass({
+  propTypes: {
+    children: React.PropTypes.node,
+    color: React.PropTypes.string,
+    icon: React.PropTypes.string,
+    styles: React.PropTypes.object,
+    title: React.PropTypes.string
+  },
+
+  getInitialState () {
+    return {
+      isOpen: false
```

Comment:
hmm good question. 

On that note, since there is a typical 'message box content' [spec](https://app.frontify.com/d/KHQZnlj1XMIO/web-patterns#/general/messaging) i'm planning on creating a component for that that we can use for default children, but then if we want to pass different children we have that option. what do you think of that idea?